source/documentation/upgrade_guide/chap-Updates_between_Minor_Releases.adoc

```diff
@@ -9,6 +9,20 @@ ifdef::context[:parent-context: {context}]
 
 To update from your current version of {vername_rhv} to the latest version of {vername_rhv}, update the {engine-name},  update the hosts, and then change the compatibility version for the cluster, virtual machines, and data center.
 
+ifdef::rhv-doc[]
+[NOTE]
+====
+If upgrading from version 4.4.9 to a later version fails on RHV-H, run the `dnf reinstall redhat-virtualization-host-image-update` command to fix the issue.
+====
+endif::rhv-doc[]
+
+ifdef::ovirt-doc[]
+[NOTE]
+====
+If upgrading from version 4.4.9 to a later version fails on RHV-H, run the `dnf reinstall ovirt-node-ng-image-update` command to fix the issue.
```

Comment:
for oVirt - I think this needs to say oVirt Node - but you can combine both notes without any conditions by simply replacing with the attribute {hypervisor-shortname}

Reply:
Kept the ifdef because the command syntax is different for RHV and oVirt but replaced RHV-H with  {hypervisor-shortname} as requested.